fileversion.go

```diff
@@ -169,32 +171,37 @@ type FixedFileInfo struct {
 	FileSubtype    VerFileSubtype
 }
 
-type StringFileInfoNames string
+type FileVersionString string
 
 const (
-	FileInfoNameComments         StringFileInfoNames = "Comments"
-	FileInfoNameCompanyName      StringFileInfoNames = "CompanyName"
-	FileInfoNameFileDescription  StringFileInfoNames = "FileDescription"
-	FileInfoNameFileVersion      StringFileInfoNames = "FileVersion"
-	FileInfoNameInternalName     StringFileInfoNames = "InternalName"
-	FileInfoNameLegalCopyright   StringFileInfoNames = "LegalCopyright"
-	FileInfoNameLegalTrademarks  StringFileInfoNames = "LegalTrademarks"
-	FileInfoNameOriginalFilename StringFileInfoNames = "OriginalFilename"
-	FileInfoNameProductName      StringFileInfoNames = "ProductName"
-	FileInfoNameProductVersion   StringFileInfoNames = "ProductVersion"
-	FileInfoNamePrivateBuild     StringFileInfoNames = "PrivateBuild"
-	FileInfoNameSpecialBuild     StringFileInfoNames = "SpecialBuild"
+	FileVersionComments         FileVersionString = "Comments"
+	FileVersionCompanyName      FileVersionString = "CompanyName"
+	FileVersionFileDescription  FileVersionString = "FileDescription"
+	FileVersionFileVersion      FileVersionString = "FileVersion"
+	FileVersionInternalName     FileVersionString = "InternalName"
+	FileVersionLegalCopyright   FileVersionString = "LegalCopyright"
+	FileVersionLegalTrademarks  FileVersionString = "LegalTrademarks"
+	FileVersionOriginalFilename FileVersionString = "OriginalFilename"
+	FileVersionProductName      FileVersionString = "ProductName"
+	FileVersionProductVersion   FileVersionString = "ProductVersion"
+	FileVersionPrivateBuild     FileVersionString = "PrivateBuild"
+	FileVersionSpecialBuild     FileVersionString = "SpecialBuild"
 )
 
-type FileInfoTranslation struct {
-	Language uint16
-	CodePage uint16
+type FileVersionTranslation struct {
+	Language uint
+	CodePage uint
```

Comment:
Should use `Language` and `uint` in what we expose to the user and `uint16` internally.